src/collatzConjecture.js

```diff
@@ -0,0 +1,15 @@
+export default function collatzConjecture(n){
+	if(n > 0){
+		var array = [n]
+		while(n > 1){
+			if (n % 2 === 0){
```

Comment:
the spacing between the if and the open parenthesis should be consistent with everything else. I believe it would be a lot easier to read if were all spaced out. Overall this code works well! Good job!